config.lib

```diff
@@ -2111,8 +2111,7 @@ check_osx_sdk() {
 			return 1
 		fi
 
-		# Set minimum version to 10.4 as that's when kCGBitmapByteOrder32Host was introduced
-		sysroot="-isysroot $osx_sdk_path -Wl,-syslibroot,$osx_sdk_path -mmacosx-version-min=10.4"
+		sysroot="-isysroot $osx_sdk_path -Wl,-syslibroot,$osx_sdk_path"
```

Comment:
This still needs to specify a minimum version of 10.9, which is the current (intended) minimum supported by the code.

Reply:
Ah, okay — fixed!